juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -164,13 +164,13 @@ healthbot {
                  */				
                 term above-low-threshold {
                     when {
-                        greater-than "$rpm-percent" "$low-threshold";
-                        less-than-or-equal-to "$rpm-percent" "$high-threshold";
+                        greater-than-or-equal-to "$rpm-percent" "$low-threshold";
+                        less-than "$rpm-percent" "$high-threshold";
                     }				
                     then {
                         status {
                             color yellow;
-                            message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than $rpm-low-threshold";
+                            message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than or equal to $rpm-low-threshold";
```

Comment:
There is 'rpm-percent-anomaly' field. We should take care of all the scenarios which include anomaly conditions as well

Reply:
Updated Anomaly condition in the terms.